packages/editor-ui/src/plugins/codemirror/completions/datatype.completions.ts

```diff
@@ -541,6 +552,29 @@ const booleanOptions = (): Completion[] => {
 	});
 };
 
+const isWithinMargin = (ts: number, now: number, margin: number): boolean => {
+	return ts > now - margin && ts < now + margin;
+};
+
+const isPlausableTimestamp = (ts: number): false | 'ms' | 's' | 'us' => {
```

Comment:
Changed the naming, but left the return type, I like having it here when I or someone else changes the implementation it would tell me that I'm returning something incorrect